### ISSUE TITLE
Russian warlord unifiers - One unifier suddenly get annexed despite the capitulation is not 100% during the final reunification war

### ISSUE THREAD

**Issue by Ikidikid** [TNODevs/TNO-Bug-Tracker#3123]
### Operating System

Windows 10 Enterprise

### HOI4 version

1.14.4

### TNO version

1.5.0c

### Do you have other mods in use?

None

### Were you using Steam?

Yes

### Which major expansions do you NOT have?

I have all

### Files

[Logs.zip](https://github.com/TNODevs/TNO-Bug-Tracker/files/14884219/Logs.zip)
[USA_play.zip](https://github.com/TNODevs/TNO-Bug-Tracker/files/14884226/USA_play.zip) Run the game to see what happen

### Describe the bug

When the final Battle for Russia starts, one unifier suddenly get annexed with the less being occupied despite the capitulation is not 100%.

### Steps to reproduce / Cause of the issue

1. Only two Russian warlord unifiers are left in Russia region.
2. Final Battle for Russia triggers.

### Expected behaviour

I don't know.

### Screenshots

![Image_1](https://github.com/TNODevs/TNO-Bug-Tracker/assets/131833099/b8ef48c9-234f-424f-9012-1f2b74ac0beb)

![Image_2](https://github.com/TNODevs/TNO-Bug-Tracker/assets/131833099/9bd9c297-6c4f-403d-a575-e55a0c1ccab5)

**Comment by ThePikminCaptain** [comment]
Yeltsin just got Petlined

Jokes aside, ima guess and say somehow peaceful unification happened at the same time

**Comment by Jack121Q** [comment]
Yes, there is a system to automatically capitulate a warlord if it the war ends too long so as to not break the sequence of events for Russia.
